src/main/java/seedu/recipe/storage/ExportManager.java

```diff
@@ -0,0 +1,87 @@
+package seedu.recipe.storage;
+
+
+import javax.swing.*;
+import javax.swing.filechooser.FileNameExtensionFilter;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+public class ExportManager {
+
+    private final Path recipeBookFilePath = Paths.get("data", "recipebook.json");
+
+    private boolean isOverwrite;
+
+    public ExportManager() {
+        this.isOverwrite = false;
+    }
+
+    public void execute() throws IOException {
+        JFileChooser chooser = createFile();
+        int result = chooser.showSaveDialog(null);
+        if (result == JFileChooser.APPROVE_OPTION) {
+            if (exists(chooser)) {
+                int response = JOptionPane.showConfirmDialog(null,
+                        "The file " + chooser.getSelectedFile().getName() + " already exists. " +
```

Comment:
use String.format and a constant here to make it neater

Reply:
Done